src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -197,7 +196,7 @@ export function createLineOrScatterChartRuntime(
 ): {
   chartJsConfig: ChartConfiguration;
   background: Color;
-  dataSetsValues: DatasetValues[];
+  dataSetsValues: DatasetValues[]; // ADRM TODO uh ? whu return this
```

Comment:
you're the one who introduced it with the scatter chart ;-)

Reply:
Oh I put the comment and forgot in investigate sorry :sweat_smile: 

Yeah this was something thatI introduced, but that is not needed anymore since some of Anthony's chart refactoring. I added a commit to remove that (-300 lines of snapshot :eyes: )